wordpress/content.md

```diff
@@ -36,3 +37,8 @@ container's IP, standard port mappings can be used:
 
 Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a
 browser.
+
+If you'd like to use an external database instead of a linked `mysql` container
```

Comment:
I just noticed there should probably be a comma at the end of this line (after "container"). Should I amend this commit?